roles/confluent.kafka_connect/tasks/main.yml

```diff
@@ -3,6 +3,15 @@
     name: confluent.common
   when: not common_role_completed|bool
 
+- name: Gather OS Facts
+  setup:
+    filter: "{{item}}"
+    gather_subset:
```

Comment:
Might be worth leaving a comment about how we only gather a specific set of facts, and we ignore the rest.